server/service/src/sync/remote_data_synchroniser.rs

```diff
@@ -229,14 +240,11 @@ impl RemoteDataSynchroniser {
     }
 }
 
-pub(crate) fn get_push_cursor(connection: &StorageConnection) -> Result<u64, RepositoryError> {
-    let value =
-        KeyValueStoreRepository::new(connection).get_i32(KeyValueType::RemoteSyncPushCursor)?;
-    let cursor = value.unwrap_or(0);
-    Ok(cursor as u64)
-}
-
-fn update_push_cursor(connection: &StorageConnection, cursor: u64) -> Result<(), RepositoryError> {
-    KeyValueStoreRepository::new(connection)
-        .set_i32(KeyValueType::RemoteSyncPushCursor, Some(cursor as i32))
+impl From<PushSyncRecord> for RemoteSyncRecordV5 {
```

Comment:
I think it's ok to use From when it's used like RemoteSyncRecordV5::from (because can navigate here with click through, rather then the trait)